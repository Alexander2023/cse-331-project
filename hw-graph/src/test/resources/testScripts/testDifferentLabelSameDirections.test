# Create a graph
CreateGraph graph1

# Add a pair of nodes
AddNode graph1 n1
AddNode graph1 n2

# Add two edges with different labels in same direction
AddEdge graph1 n1 n2 e1
AddEdge graph1 n1 n2 e2

# Print the children of the nodes
ListChildren graph1 n1
ListChildren graph1 n2