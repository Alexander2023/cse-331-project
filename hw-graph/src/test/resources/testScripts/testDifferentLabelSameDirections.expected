# Create a graph
created graph graph1

# Add a pair of nodes
added node n1 to graph1
added node n2 to graph1

# Add two edges with different labels in same direction
added edge e1 from n1 to n2 in graph1
added edge e2 from n1 to n2 in graph1

# Print the children of the nodes
the children of n1 in graph1 are: n2(e1) n2(e2)
the children of n2 in graph1 are: